projects/ui-components/lf-selection-list/lf-selection-list.component.ts

```diff
@@ -386,14 +385,16 @@ export class LfSelectionListComponent implements AfterViewInit, OnDestroy {
         }
         this.ref.detectChanges();
 
-        const shouldFillLastColumn = this.allColumnDefs[this.allColumnDefs.length - 1]?.defaultWidth === 'auto';
-        if (!shouldFillLastColumn) {
-          const containerWidth = this.viewport?.elementRef.nativeElement.getBoundingClientRect().width;
-          tableEl.style.width = containerWidth + 'px';
-          this.ref.detectChanges();
-          const widthsInPixel: string[] = [];
+        const containerWidth = this.viewport?.elementRef.nativeElement.getBoundingClientRect().width;
+        tableEl.style.width = containerWidth + 'px';
+        this.ref.detectChanges();
+        const widthsInPixel: string[] = [];
 
-          this.allColumnDefs.forEach((col) => {
+        this.allColumnDefs.forEach((col, index) => {
+          const lastColumnWidthAuto = index === this.allColumnDefs.length - 1 && col.defaultWidth === 'auto';
```

Comment:
should only leave the column's width as auto if there is only one column (the name)